concent_api/core/models.py

```diff
@@ -296,7 +296,7 @@ class Meta:
 
     # Related messages
     task_to_compute             = OneToOneField(StoredMessage, blank = True, null = False, related_name = 'subtasks_for_task_to_compute')
-    report_computed_task        = OneToOneField(StoredMessage, blank = True, null = True, related_name = 'subtasks_for_report_computed_task')
+    report_computed_task        = OneToOneField(StoredMessage, blank = True, null = False, related_name = 'subtasks_for_report_computed_task')
```

Comment:
If it's non-nullable then there's no reason to allow `blank` here.

Reply:
corrected for both fields